common/xbps-src/shutils/chroot.sh

```diff
@@ -106,11 +106,23 @@ chroot_prepare() {
         msg_error "Bootstrap not installed in $XBPS_MASTERDIR, can't continue.\n"
     fi
 
-    # Create some required files.
-    if [ -f /etc/localtime ]; then
-        cp -f /etc/localtime $XBPS_MASTERDIR/etc
-    elif [ -f /usr/share/zoneinfo/UTC ]; then
-        cp -f /usr/share/zoneinfo/UTC $XBPS_MASTERDIR/etc/localtime
+    # Setup timezone information
+    #
+    # According to https://www.freedesktop.org/software/systemd/man/localtime.html:
+    #
+    #   Because the timezone identifier is extracted from the symlink target
+    #   name of /etc/localtime, this file may not be a normal file or hardlink.
+    #
+    # So we cannot just copy /etc/localtime into $XBPS_MASTERDIR. Instead, copy
+    # over the zoneinfo for UTC and create the /etc/localtime symlink. 
```

Comment:
Maybe shorten the explanation to
> Some software expects /etc/localtime to be a symbolic link it can read to determine the name of the time zone, so set up the expected link structure.

Reply:
Done.